Exploration of the MNIST1000 Dataset.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
<p>interesting observations, but can you show the data/visualizations which support these findings?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>